configargparse.py

```diff
@@ -241,6 +241,18 @@ def parse_known_args(self, args = None, namespace = None,
         else:
             args = list(args)
 
+        new_args = list()
```

Comment:
would be good to replace `new_args` with a more specific name like `normalized_args`